app/models/enterprise.rb

```diff
@@ -15,6 +15,9 @@ class Enterprise < ApplicationRecord
     medium: { resize_to_fill: [720, 156] },
     large: { resize_to_fill: [1200, 260] },
   }.freeze
+  WHITE_LABEL_LOGO_SIZES = {
+    default: { gravity: "Center", resize: "217x44^", crop: '217x44+0+0' }
```

Comment:
I can see this is the rendered size of the existing logo on desktop, so it seems like the ideal resolution to store.

Many screens can display at higher density (eg macbook and most phones). But it can be a bit of a rabbit-hole trying to optimise all this so let's just stick with this for now ;)

Reply:
> `# The next Rails version will have named variants`

It could have been good to try that out 🤷

Reply:
Changed! 2b0d3a6134